mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java

```diff
@@ -14,25 +14,35 @@
 
 package com.google.mediapipe.apps.instantmotiontracking;
 
-import java.util.ArrayList;
 import com.google.protobuf.*;
```

Comment:
This is not encouraged. Can you avoid the * and specify which one you are importing?